bin/config.ts

```diff
@@ -3,8 +3,8 @@ import { BillingMode } from 'aws-cdk-lib/aws-dynamodb';
 import { TableCapacityConfig } from './stacks/cron-stack';
 
 export const PROD_TABLE_CAPACITY: TableCapacityConfig = {
-  fillerAddress: { billingMode: BillingMode.PROVISIONED, readCapacity: 20, writeCapacity: 100 },
+  fillerAddress: { billingMode: BillingMode.PROVISIONED, readCapacity: 50, writeCapacity: 200 },
```

Comment:
Should we bump it up more? In the chart you shared, it does cross these new thresholds too

Reply:
hmm yeah I guess it doesn't really matter if we over-provision for a bit